activesupport/test/cache/behaviors/cache_increment_decrement_behavior.rb

```diff
@@ -11,7 +11,9 @@ def test_increment
     assert_equal 3, @cache.read(key, raw: true).to_i
 
     missing = @cache.increment(SecureRandom.alphanumeric)
-    assert(missing.nil? || missing == 1)
+    assert(missing == 1)
```

Comment:
The code was using a raw `assert` to be able to have this `||`, you can use `assert_equal` now.
```suggestion
    assert_equal 1, missing
```